js/style/style.js

```diff
@@ -731,7 +732,7 @@ Style.prototype = util.inherit(Evented, {
 
     // Callbacks from web workers
 
-    'get sprite json': function(params, callback) {
+    'get sprite json': function(_, params, callback) {
```

Comment:
Why did you choose to pass `mapId` to the `style` callbacks? It doesn't seem to be used anywhere.